code_examples/sdk_examples/src/dapp/dapp/domainLinkageCtype.ts

```diff
@@ -1,16 +1,29 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
 
-export const domainLinkageCType: Kilt.ICType = {
-  $id: 'kilt:ctype:0x9d271c790775ee831352291f01c5d04c7979713a5896dcf5e81708184cc5c643',
-  $schema: 'http://kilt-protocol.org/draft-01/ctype#',
-  title: 'Domain Linkage Credential',
-  properties: {
-    id: {
-      type: 'string'
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export async function main(): Promise<Kilt.ICType> {
```

Comment:
Why don't you call it `getDomainLinkageCType` or just `domainLinkageCType`?

Reply:
I was following the example of other snippets, which called this function main. I don't know what the convention is here tbh